app/src/main/java/ca/bc/gov/bchealth/ui/services/ServicesScreen.kt

```diff
@@ -0,0 +1,256 @@
+import ca.bc.gov.common.model.services.OrganDonationStatus
+
+@Composable
+fun ServicesScreen(
+    modifier: Modifier,
+    viewModel: ServicesViewModel,
+    onRegisterOnUpdateDecisionClicked: (String) -> Unit,
+    onDownloadButtonClicked: (String) -> Unit
+) {
+    val uiState = viewModel.uiState.collectAsState()
+
+    ServiceScreenContent(
+        modifier = modifier,
+        onLoading = uiState.value.onLoading,
+        organDonationRegistrationDetail = uiState.value.organDonationRegistrationDetail,
+        onRegisterOnUpdateDecisionClicked = { url ->
+            onRegisterOnUpdateDecisionClicked(url)
+        }
+    ) {
+        uiState.value.organDonationRegistrationDetail?.file?.let { file ->
+            onDownloadButtonClicked(
+                file
+            )
+        }
+    }
+}
+
+@Composable
+private fun ServiceScreenContent(
+    modifier: Modifier = Modifier,
+    onLoading: Boolean,
+    organDonationRegistrationDetail: OrganDonationRegistrationDetail?,
+    onRegisterOnUpdateDecisionClicked: (String) -> Unit,
+    onDownloadButtonClicked: () -> Unit
+) {
+
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .padding(start = 32.dp, end = 32.dp, bottom = 32.dp)
+    ) {
+
+        Text(
+            text = stringResource(id = R.string.services),
+            style = MyHealthTypography.h2,
+            color = MaterialTheme.colors.primary
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        Text(
+            modifier = Modifier.fillMaxWidth(),
+            text = stringResource(id = R.string.services_subtitle),
+            style = MyHealthTypography.h4
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+
+        if (onLoading) {
+            Box(
+                modifier = modifier
+                    .fillMaxSize()
+            ) {
+                CircularProgressIndicator(
+                    modifier = Modifier.align(Alignment.Center),
+                )
+            }
+        } else {
+            organDonationRegistrationDetail?.let {
+                OrganDonor(it, onRegisterOnUpdateDecisionClicked = { url ->
+                    onRegisterOnUpdateDecisionClicked(url)
+                }) {
+                    onDownloadButtonClicked()
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun OrganDonor(
+    organDonationRegistrationDetail: OrganDonationRegistrationDetail,
+    onRegisterOnUpdateDecisionClicked: (String) -> Unit,
+    onDownloadButtonClicked: () -> Unit
+) {
+
+    Card(
+        modifier = Modifier
+            .fillMaxWidth(),
+        shape = MaterialTheme.shapes.medium,
+        elevation = 4.dp,
+    ) {
+        Column(modifier = Modifier.padding(24.dp)) {
+            Row(verticalAlignment = Alignment.Bottom) {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_organ_donor),
+                    contentDescription = "organ donor",
```

Comment:
Could you move these contentDescriptions in this file to the strings file?

Reply:
done